### PR TITLE
Migrate 'how to run a hackathon' to handbook

### PR DESCRIPTION
**Pull request by jasonblais**
Related docs PR: https://github.com/mattermost/docs/pull/3583

PS: No full editor review needed as this is just a copy-paste of existing content.

### CODE REVIEW

**Review by justinegeffen — DISMISSED** [review]

File: contributors/contributors/how-to-run-a-hackathon.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+2. Identify 1 or 2 members in marketing to partner with, who can help with tweets, blog posts and other marketing content to generate awareness of the Hackathon.
+3. Identify 1 or 2 members in ops to partner with, who can help with meeting scheduling, SWAG and other operational tasks.
+
+# C. (T-minus 25 working days)
+
+1. Work with ops to decide on what SWAG to send for participation - both for contributors and staff.
+2. Create a landing page for the Hackathon, which contains all information and to which other communication items are linked to.
+
+  - This can be a webpage such as [one for Feb/2019 Hackfest](https://mattermost.com/hackfest2019/) or a GitHub repository such as [one for Nov/2019 Hackathon](https://github.com/mattermost/mattermost-hackathon-nov2019).
+  - All processes and documentations should be clearly included on this landing page. This should include process for finding and assigning teams, and submission guidelines. FAQs can be incorporated as needed.
```
Comment (justinegeffen) [inline contributors/contributors/how-to-run-a-hackathon.md]:
```suggestion
  - All processes and documentation should be clearly included on this landing page, including a process for finding and assigning teams and project submission guidelines. FAQs can be incorporated as needed.
```

File: contributors/contributors/how-to-run-a-hackathon.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
+# How to run a Hackathon
+
+This page outlines best practices and recommended steps for running a Hackathon.
+
+Previously, Hackathons have run twice a year with one coinciding with MatterCon. In future, Hackathons may be run 3 or 4 times a year.
```
Comment (justinegeffen) [inline contributors/contributors/how-to-run-a-hackathon.md]:
```suggestion
Previously, Hackathons have run twice a year with one coinciding with MatterCon. In future, Hackathons may be run three or four times a year.
```

File: contributors/contributors/how-to-run-a-hackathon.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+T-minus counts measure "working days" (weekdays other than major holidays concurrent in US and Canada) prior to the Hackathon day. All times are based on 8:00am ([Palo Alto Time](http://everytimezone.com/)) unless otherwise stated.
+
+## A. (T-minus 40 working days)
+
+1. Decide dates for the Hackathon. Some things to take into consideration:
+
+  - Avoid overlap with major holidays in US, Canada, Europe or Asia (including Australia and Japan).
```
Comment (justinegeffen) [inline contributors/contributors/how-to-run-a-hackathon.md]:
```suggestion
  - Avoid overlap with major holidays in US, Canada, Europe, or Asia (including Australia and Japan).
```

File: contributors/contributors/how-to-run-a-hackathon.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+  - Avoid overlap with significant release dates, such as release testing or feature complete. Recommended time is starting a Hackathon within a week after feature complete.
+  - Avoid overlap with significant company dates that take significant amount of developers' time, such as performance evaluations.
+  - Avoid overlap with significant community dates, such as major developer conferences/events.
+  - Avoid starting on a Friday - this results in a kick-off call on a late evening for Europe/Asia, effectively meaning the Hackathon starts on a weekend in those areas.
```
Comment (justinegeffen) [inline contributors/contributors/how-to-run-a-hackathon.md]:
```suggestion
  - Avoid starting on a Friday - this results in a kick-off call in the late evening for Europe/Asia, effectively meaning the Hackathon starts on a weekend in those areas.
```

File: contributors/contributors/how-to-run-a-hackathon.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+  - Previous Hackathons ran for 2 days with presentations on the third date. Last time developers suggested adding a third date to allow more time for polish and increased quality.
```
Comment (justinegeffen) [inline contributors/contributors/how-to-run-a-hackathon.md]:
```suggestion
  - Previous Hackathons ran for two days with presentations on the third date. At the end of the most recent Hackathon participating developers suggested adding a third day to allow more time for polish and increased quality.
```

File: contributors/contributors/how-to-run-a-hackathon.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+2. Identify at least 1 or 2 developers to partner with during the Hackathon. Start by reaching out to either VP Engineering or Dev Leads to hear their recommendations.
```
Comment (justinegeffen) [inline contributors/contributors/how-to-run-a-hackathon.md]:
```suggestion
2. Identify at least one or two developers to partner with during the Hackathon. Start by reaching out to either VP Engineering or Dev Leads to hear their recommendations.
```

File: contributors/contributors/how-to-run-a-hackathon.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+
+## B. (T-minus 30 working days)
+
+1. Put together a project plan, including task description, start date, due date, owner and any additional notes. [See an example](https://docs.google.com/spreadsheets/d/15iiLPzJjlajHQVgQa9HulW9WfBauIlnkCOJ07oKz7_U).
```
Comment (justinegeffen) [inline contributors/contributors/how-to-run-a-hackathon.md]:
```suggestion
1. Put together a project plan, including task description, start date, due date, owner, and any additional notes. [See an example](https://docs.google.com/spreadsheets/d/15iiLPzJjlajHQVgQa9HulW9WfBauIlnkCOJ07oKz7_U).
```

File: contributors/contributors/how-to-run-a-hackathon.md [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+2. Identify 1 or 2 members in marketing to partner with, who can help with tweets, blog posts and other marketing content to generate awareness of the Hackathon.
```
Comment (justinegeffen) [inline contributors/contributors/how-to-run-a-hackathon.md]:
```suggestion
2. Identify one or two members in Marketing to partner with, who can help with tweets, blog posts, and other marketing content to generate awareness of the Hackathon.
```

File: contributors/contributors/how-to-run-a-hackathon.md [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+3. Identify 1 or 2 members in ops to partner with, who can help with meeting scheduling, SWAG and other operational tasks.
```
Comment (justinegeffen) [inline contributors/contributors/how-to-run-a-hackathon.md]:
```suggestion
3. Identify one or two members in Ops to partner with, who can help with meeting scheduling, SWAG, and other operational tasks.
```

File: contributors/contributors/how-to-run-a-hackathon.md [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+
+# C. (T-minus 25 working days)
+
+1. Work with ops to decide on what SWAG to send for participation - both for contributors and staff.
```
Comment (justinegeffen) [inline contributors/contributors/how-to-run-a-hackathon.md]:
```suggestion
1. Work with Ops to decide on what SWAG to send for participation - both for contributors and staff.
```

File: contributors/contributors/how-to-run-a-hackathon.md [hunk 11]
```diff
@@ -0,0 +1,101 @@
...
+2. Create a landing page for the Hackathon, which contains all information and to which other communication items are linked to.
```
Comment (justinegeffen) [inline contributors/contributors/how-to-run-a-hackathon.md]:
```suggestion
2. Create a landing page for the Hackathon, which contains all information and to which other communication items are linked.
```

File: contributors/contributors/how-to-run-a-hackathon.md [hunk 12]
```diff
@@ -0,0 +1,101 @@
...
+
+  - This can be a webpage such as [one for Feb/2019 Hackfest](https://mattermost.com/hackfest2019/) or a GitHub repository such as [one for Nov/2019 Hackathon](https://github.com/mattermost/mattermost-hackathon-nov2019).
+  - All processes and documentations should be clearly included on this landing page. This should include process for finding and assigning teams, and submission guidelines. FAQs can be incorporated as needed.
+
+3. Create a GitHub repository for submitting Hackathon projects. This can be the same as the landing page if a GitHub repository is used for it.
+
+# D. (T-minus 20 working days)
+
+1. Announce the Hackathon internally by posting in [Private Staff Discussion channel](https://community.mattermost.com/private-core/channels/platform) and sharing on R\&D weekly meeting.
```
Comment (justinegeffen) [inline contributors/contributors/how-to-run-a-hackathon.md]:
```suggestion
1. Announce the Hackathon internally by posting in [Private Staff Discussion channel](https://community.mattermost.com/private-core/channels/platform) and sharing in the R&D weekly meeting.
```

File: contributors/contributors/how-to-run-a-hackathon.md [hunk 13]
```diff
@@ -0,0 +1,101 @@
...
+2.  Schedule a kick-off call with all Hackathon stakeholders, including
+    the developer, marketing and ops members identified previously.
```
Comment (justinegeffen) [inline contributors/contributors/how-to-run-a-hackathon.md]:
```suggestion
    the developer, Marketing, and Ops members identified previously.
```

File: contributors/contributors/how-to-run-a-hackathon.md [hunk 14]
```diff
@@ -0,0 +1,101 @@
...
+3.  Define and set up metrics to measure impact of the Hackathon. Examples include: number of participants (staff + community separately), number of submissions, number of submissions shipped as a core feature or as a plugin in the marketplace.
```
Comment (justinegeffen) [inline contributors/contributors/how-to-run-a-hackathon.md]:
```suggestion
3.  Define and set up metrics to measure impact of the Hackathon. Examples include: Number of participants (staff + community separately), number of submissions, number of submissions shipped as a core feature or as a plugin in the Marketplace.
```

File: contributors/contributors/how-to-run-a-hackathon.md [hunk 15]
```diff
@@ -0,0 +1,101 @@
...
+
+# E. (T-minus 15 working days)
+
+1. Create Meetup links for Hackathon idea presentation, kick-off and demos. For examples, see [Mattermost Meetup page](https://www.meetup.com/mattermost/).
```
Comment (justinegeffen) [inline contributors/contributors/how-to-run-a-hackathon.md]:
```suggestion
1. Create Meetup links for Hackathon idea presentation, kick-off, and demos. For examples, see [Mattermost Meetup page](https://www.meetup.com/mattermost/).
```

File: contributors/contributors/how-to-run-a-hackathon.md [hunk 16]
```diff
@@ -0,0 +1,101 @@
...
+2. Share the Hackathon externally via Reddit, GitHub, Discourse and the [community server](https://community.mattermost.com). For examples, see previous [Reddit post](https://www.reddit.com/r/Mattermost/comments/dvvsm4/mattermost_hackathon_nov_22_26/), and [GitHub issue](https://github.com/mattermost/mattermost-server/issues/13087).
```
Comment (justinegeffen) [inline contributors/contributors/how-to-run-a-hackathon.md]:
```suggestion
2. Share the Hackathon externally via Reddit, GitHub, Discourse, and the [community server](https://community.mattermost.com). For examples, see previous [Reddit post](https://www.reddit.com/r/Mattermost/comments/dvvsm4/mattermost_hackathon_nov_22_26/), and [GitHub issue](https://github.com/mattermost/mattermost-server/issues/13087).
```

File: contributors/contributors/how-to-run-a-hackathon.md [hunk 17]
```diff
@@ -0,0 +1,101 @@
...
+
+1. Hold a Hackathon idea presentation call, scheduled for 60 minutes, with an additional 30 minutes for extra time. This time is used to gather feedback about ideas and to attract others to work as a team. To see previous slides and meeting recording, see [Hackathon Nov/2019 Google Drive](https://drive.google.com/drive/folders/1iBVGDzdwZ72oEY6mshzjcgJQ4NdhXRfW).
+
+# H. (T-minus 0 working days)
+
+1. Hold a Hackathon kick-off call, scheduled for 30 minutes. To see previous slides and meeting recording, see [Hackathon Nov/2019 Google Drive](https://drive.google.com/drive/folders/1iBVGDzdwZ72oEY6mshzjcgJQ4NdhXRfW).
+
+# I. (T-plus 3 working days)
+
+1. Hold a Hackathon demo call, scheduled for 2 hours, with an additional hour for extra time. This time is used for participants to present their project. Invite the entire Mattermost company to the call as optional attendee.
```
Comment (justinegeffen) [inline contributors/contributors/how-to-run-a-hackathon.md]:
```suggestion
1. Hold a Hackathon demo call, scheduled for 2 hours, with an additional hour for extra time. This time is used for participants to present their project. Invite the entire Mattermost organization to the call as optional attendees.
```

File: contributors/contributors/how-to-run-a-hackathon.md [hunk 18]
```diff
@@ -0,0 +1,101 @@
...
+
+  - Provide guidelines and best practices for creating presentations (both for staff and community).
+  - Three minutes allowed per Hackathon participant. For multiple submissions, five minutes total is allowed to present.
```
Comment (justinegeffen) [inline contributors/contributors/how-to-run-a-hackathon.md]:
```suggestion
  - Three minutes allowed per Hackathon participant. For participants with multiple submissions, five minutes total is allowed for the presentations.
```

File: contributors/contributors/how-to-run-a-hackathon.md [hunk 19]
```diff
@@ -0,0 +1,101 @@
...
+  - Ask participants who can't attend to submit a demo video, which are shared during the call.
```
Comment (justinegeffen) [inline contributors/contributors/how-to-run-a-hackathon.md]:
```suggestion
  - Ask participants who can't attend to submit a demo video, which is shared during the call.
```

**Comment by jasonblais** [comment]
@justinegeffen Thank you! All suggestions applied

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**